plugins/org.jkiss.dbeaver.tasks.ui.view/plugin.xml

```diff
@@ -58,7 +58,10 @@
         </definition>
         <definition id="org.jkiss.dbeaver.tasks.editable">
             <with variable="activePart">
-                <test property="org.jkiss.dbeaver.core.global.canCreateConnection"/>
+                <and>
+                    <test property="org.jkiss.dbeaver.core.global.canCreateConnection"/>
+                    <test property="org.jkiss.dbeaver.core.global.hasPermission" value="database-developer"/>
```

Comment:
Let's move poermission check into Java. canCreateConnection must return false if user don't have this permission.